plugin/storage/factory_config.go

```diff
@@ -29,13 +30,21 @@ const (
 	DependencyStorageTypeEnvVar = "DEPENDENCY_STORAGE_TYPE"
 
 	spanStorageFlag = "--span-storage.type"
+
+	// DownSamplingRatio is the name of the env var that defines the ratio of spans we would drop
+	DownSamplingRatio = "DOWN_SAMPLING_RATIO"
+
+	// DownSamplingHashSalt is the name of the env var that stores the hash salt for down sampling
+	DownSamplingHashSalt = "DOWN_SAMPLING_HASH_SALT"
```

Comment:
nit: downsampling is usually spelled as a single word, so we can replace `DownSampling` with `Downsampling`.